### ISSUE TITLE
Question - Thingspeak

### ISSUE THREAD

**Issue by gnomosg** [CongducPham/LowCostLoRaGw#221]
Hi, 
I try to upload data to multiple channels using  CloudThingSpeak.py
in line 184
_for k in (0, len(key_ThingSpeak.key_association)-1):_
I think it should be:
_for k in range(0, len(key_ThingSpeak.key_association)-1):_
to loop over all values in list...
Could you check this?
Regards and thanks for great work :)

**Comment by CongducPham** [comment]
Yes definitely, actually I forgot to change it on the github version. Will do it Monday. Regards.
